resources/ts/Components/IntegrationCard.tsx

```diff
@@ -68,9 +68,14 @@ export const IntegrationCard = ({ id, name, type, status, onEdit }: Props) => {
               <ButtonIconCopy onClick={handleCopyToClipboard} />
             </Tooltip>
           </div>
-            {type === "widgets" && (
-                <ButtonLinkSecondary href={'/integrations/' + id + '/widget'} target="_blank"> {t("integrations.open_widget")} </ButtonLinkSecondary>
-            )}
+          {type === "widgets" && (
+            <ButtonLinkSecondary
+              href={"/integrations/" + id + "/widget"}
```

Comment:
It's a question of preference.
But you could also use template literals here.
```suggestion
              href={`/integrations/${id}/widget`}
```